.github/workflows/cd-dgraph-js-http.yml

```diff
@@ -64,9 +64,10 @@ jobs:
         with:
           node-version: '20.x'
           registry-url: 'https://registry.npmjs.org'
+          scope: '@dgraph-io'
       - name: Build dgraph-js-http package
         run: npm ci --legacy-peer-deps
       - run: npm publish
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_DGRAPH_JS_HTTP_TOKEN }}
```

Comment:
Hey Josh just checking, is this change to the token secret name intended as well?

Reply:
Hey Kevin - yes I think it can be good to use granular access tokens here, so the name can be more specific

Reply:
Got it!